src/functions-and-arrays.js

```diff
@@ -29,9 +100,29 @@ const wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray (wordsUnique) {
```

Comment:
Este es complejito. Hay varias soluciones haciendo la búsqueda de "how to remove duplicates from array in javascript"